application/classes/Ushahidi/Validator/Tag/Update.php

```diff
@@ -54,7 +54,8 @@ protected function getRules()
 				['color'],
 			],
 			'icon' => [
-				['alpha_dash'],
+				// alphas, numbers, punctuation, and spaces
```

Comment:
punctuation? I can see `icon-names` and `icon something` .. but other punctuation?